### DISCUSSION TITLE
Retaining earth wall

### DISCUSSION THREAD

**Discussion by tinnishill** [OpenOrienteering/mapper#2278] [Questions & Answers]
I  have just noticed, on the IOF Map Wiki, that ISOM2017-2/v6 now contains a symbol 105.2  , "retaining earth wall". It could be useful for my current project. Any ideas about a timetable for introduction ?

**Comment by dg0yt** [comment]
Updated symbol sets exist, in issues/PRs here and on my disk. I'm optimistic to get something merged this year. 
However, integrating and validating whole symbol sets, color definitions and migration paths is really tedious. Several attempts are unfinished at the moment.

**Comment by tinnishill** [comment]
Thanks for the information. It's good of you to reply.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
